### ISSUE TITLE
Support for Duration "part" methods

### ISSUE THREAD

**Issue by MMauro94** [js-joda/js-joda#471]
Hi, would it be possible to add the following methods to the `Duration` class:
* `toDaysPart()`
* `toHoursPart()`
* `toMinutesPart()`
* `toSecondsPart()`
* `toNanosPart()`

They should mimic the behavior of [java.time.Duration](https://docs.oracle.com/javase/9/docs/api/java/time/Duration.html) (they extract the "part" information of the respective unit). Example:

```js
const d = Duration.parse("PT1H30M20S"); //1 hour, 30 minutes, 20 seconds

d.toHours(); // 1
d.toHoursPart(); // 1

d.toMinutes(); // 90
d.toMinutesPart(); // 30

d.seconds(); // 5420
d.toSecondsPart(); // 20
```

**Comment by pithu** [comment]
Yes, since it is part of the threeten API we would accept pull requests for it.
